scripts/deploy/govcms-update_site_alias

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+IFS=$'\n\t'
+set -euo pipefail
+
+#
+# GovCMS prepare for deploy.
+#
+# This performs some perfunctory commands to ensure that the
+# sites are in a deployable position.
+#
+# @deprecated 1.0
+#
+# This is for the legacy projects that have not been updated with the latest
+# scaffold
+#
+
+sed -i "s/%%PROJECT_NAME%%/\${LAGOON_PROJECT}/g" /app/drush/sites/govcms.site.yml
```

Comment:
this will fail for projects that have their alias file located in another place. Maybe worth wrapping this in `if`.